src/test/java/org/apache/ibatis/builder/ParameterExpressionTest.java

```diff
@@ -133,16 +126,20 @@ public void shouldIgnoreLeadingAndTrailingSpaces() {
 
   @Test
   public void invalidOldJdbcTypeFormat() {
-    expectedException.expect(BuilderException.class);
-    expectedException.expectMessage(is("Parsing error in {id:} in position 3"));
-    new ParameterExpression("id:");
+    try {
+      new ParameterExpression("id:");
+    } catch (BuilderException e) {
+      e.getMessage().contains("Parsing error in {id:} in position 3");
```

Comment:
Oh, and it also needs `assertTrue` or something to verify the result of `contains()` method XD